Bow OpenAPI/Generator/Support Files/Generator.m

```diff
@@ -0,0 +1,7 @@
+//  Copyright © 2019 The Bow Authors
+
+#import "Generator.h"
+
+@implementation Generator
+
+@end
```

Comment:
Why do we need this Objective-C file?

Reply:
this is from old targets, good you find it! we have to remove it :)